### PR TITLE
chore(ts): Convert adminOrganizations

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/admin/adminOrganizations.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <strong>
+      <Link to={`/${row.slug}/`}>{row.name}</Link>
+    </strong>
+    <br />
+    <small>{row.slug}</small>
+  </td>,
+];
+
+const AdminOrganizations = (props: Props) => (
+  <div>
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/admin/adminOrganizations.tsx:20]: would be fine to replace the `div` with ` React.Fragment`?

File: src/sentry/static/sentry/app/views/admin/adminOrganizations.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <h3>{t('Organizations')}</h3>
+    <ResultGrid
+      path="/manage/organizations/"
+      endpoint="/organizations/?show=all"
+      method="GET"
+      columns={[<th key="column-org">Organization</th>]}
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/admin/adminOrganizations.tsx:26]:
```suggestion
      columns={[<th key="column-org">{t('Organization')}</th>]}
```

**Review by priscilawebdev — APPROVED** [review]

File: src/sentry/static/sentry/app/views/admin/adminOrganizations.tsx (hunk 2 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/views/admin/adminOrganizations.tsx:26]: @priscilawebdev We don't really translate admin views.

File: src/sentry/static/sentry/app/views/admin/adminOrganizations.tsx (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/admin/adminOrganizations.tsx:20]: maybe 😬 not gonna look to hard since this is just an admin view

**Pull request merged**